src/commands/install-extensions.ts

```diff
@@ -136,14 +136,15 @@ async function installExtension(extension: string, sources: Record<string, Sourc
 
 		await vscode.commands.executeCommand('workbench.extensions.installExtension', extension);
 
+		installedExtensions[extension] = null;
```

Comment:
Can you replace the `null` with an empty string `''`? Thx

Reply:
Alright! Did so